flask_restplus/api.py

```diff
@@ -492,6 +492,7 @@ def handle_error(self, e):
             }
 
         default_data['message'] = default_data.get('message', str(e))
+        headers = default_data.pop("headers", headers)
```

Comment:
It wouyld be better to use  ```flask_restful.unpack``` (will be merged soon) to properly unpack response above:
```python
elif e.__class__ in self._error_handlers:
    handler = self._error_handlers[e.__class__]
    result = handler(e)
    default_data, code, headers = restful.unpack(result)
elif self._default_error_handler:
    result = self._default_error_handler(e)
    default_data, code, headers = restful.unpack(result)
```
The only problem is that `unpack` always take 200 as default status code.
If you wait a little bit, I'l merge `unpack` allowing to override the default status code and to write:
```python
if isinstance(e, HTTPException):
    code = e.code
    default_data = {
        'message': getattr(e, 'description', HTTP_STATUS_CODES.get(code, ''))
     }
     headers = e.get_response().headers
elif e.__class__ in self._error_handlers:
    handler = self._error_handlers[e.__class__]
    result = handler(e)
    default_data, code, headers = restful.unpack(result, 500)
elif self._default_error_handler:
    result = self._default_error_handler(e)
    default_data, code, headers = restful.unpack(result, 500)
else:
    code = 500
    default_data = {
        'message': HTTP_STATUS_CODES.get(code, str(e)),
    }
    headers = Headers()
```

What do you think about it ?